extensions/sonarcloud/templates/gradle/template.json

```diff
@@ -0,0 +1,109 @@
+{
+  "id": "sonarcloud_gradle_template",
```

Comment:
A build template is just a _.json_ file listing the tasks to be executed, and the property values to be pre-set for those tasks. The template file must be called _template.json_. There is no executable code.
I've added four templates: Maven, Gradle, VS Build and .NET Core build.